central/notifiers/syslog/syslog_test.go

```diff
@@ -104,6 +124,91 @@ func (s *SyslogNotifierTestSuite) TestCEFExtensionFromPairs() {
 	s.Equal(fmt.Sprintf("%s %s", extensionPair1, extensionPair2), extension)
 }
 
+func (s *SyslogNotifierTestSuite) TestValidateSyslogEmptyExtrafields() {
```

Comment:
Everyone writes tests a little differently so grain of salt, but I've become a fan of what Mandar has done with struct definitions and for loops to allow for easy expansion of other test cases. For an example of what that looks like you can check out this function: https://github.com/stackrox/stackrox/blob/1bce7ff4a2c57948e7658b1b91a627330457980e/tests/resourcecollection_test.go#L162

Reply:
In case of collections one method is tested with different inputs. This is a great way to do it
For syslog uni tests, multiple method are tested - alertotcef, audittocef, validate syslog. Thats why I wrote separate tests